tests/installation/user_settings.pm

```diff
@@ -34,13 +34,18 @@ sub run {
         return;
     }
 
-    $self->enter_userinfo();
-    assert_screen 'inst-userinfostyped';
-    if (match_has_tag 'boo#1122804') {
-        diag "trying again with slow typing due to https://progress.opensuse.org/issues/46190\n";
-        $self->enter_userinfo(retry => 1);
-        assert_screen 'inst-userinfostyped';
-    }
+    # retry if not typed correctly
+    my $max_tries = 2;
+    my $retry     = 0;
```

Comment:
you can just define a single variable `my $retries = 2;`, decrement the variable in each run and check if it's >= 0. This is also how it's done in other places.

Reply:
To do that, the function enter_userinfo needs to be change or retry will be non-zero the first time, causing slow typing on any scenario.